tools/c7n_mailer/c7n_mailer/sqs_message_processor.py

```diff
@@ -260,9 +268,9 @@ def render(self, target, data, resources):
         try:
             template = self.env.get_template("%s.j2" % (
                 data['action'].get('template', 'default')))
-        except jinja2.TemplateNotFound:
-            self.logger.error("Invalid template reference %s.j2" % (
-                data['action'].get('template', 'default')))
+        except Exception as error_msg:
+            self.logger.error("Invalid template reference %s.j2\n%s" % (
+                data['action'].get('template', 'default')), error_msg)
```

Comment:
by bubbling up the actual error from jinja, it's easier to find out what the problems are for people looking at lambda mailer logs.